item.go

```diff
@@ -106,3 +107,42 @@ func (i *Item) AddDuration(durationInSeconds int64) {
 	}
 	i.IDuration = parseDuration(durationInSeconds)
 }
+
+type ParseDurationFunc func(duration int64) string
```

Comment:
two things here...

* why are you defining a type?  the type is already inferred from the `func` definition and not needed.  did some particular linter raise a warning that I am not aware of?  (i use strict linters, and always look for that kind of stuff)
* it's upper case, which means it is exporting - and polluting the API (visible by others when they make pull requests). if it remains (by justifying my questions above), it must be lowerCased.

i think all u need to do is to remove this line, and the type from line 113.

Reply:
- I think it would be just a good practice to define how a function should behave if you would like to keep them assigned to a variable. There is no liter warning here.
- Exporting the function type makes them visible in the API but users would also be able to use their own implementation if they have a need to controlling the behavior.